app/controllers/chargeback_controller.rb

```diff
@@ -208,28 +228,48 @@ def cb_rate_edit
       if params[:typ] == "copy" # if tab was not changed
         session[:changed] = true
         @sb[:rate_details] = []
+        @sb[:num_tiers] = []
+        @sb[:tiers] = []
         rate = ChargebackRate.find(obj[0])
         @sb[:rate] = ChargebackRate.new
         @sb[:rate].description = "Copy of " + rate.description
         @sb[:rate].rate_type = rate.rate_type
-        rate_details = rate.chargeback_rate_details
+        rate_details = rate.chargeback_rate_details.to_a
+        rate_details.sort_by! { |rd| [rd.group.downcase, rd.description.downcase] }
         # Create new rate detail records for copied rate record
-        rate_details.each do |r|
+        rate_details.each_with_index do |r, i|
+          @sb[:tiers][i] = []
```

Comment:
Please don't use single character variables unless it's a 1-3 line block/method.  I don't know what `i` is much later.  Even if I guess it's an index, I don't know which index it is.